benchmarks/benchmark_throughput.py

```diff
@@ -171,10 +171,13 @@ def run_vllm(
     llm = LLM(**dataclasses.asdict(engine_args))
 
     # Add the requests to the engine.
-    prompts: List[TextPrompt] = []
+    prompts: List[TextPrompt | TokensPrompt] = []
```

Comment:
This syntax was added in python 3.10 https://peps.python.org/pep-0604/. Since 3.9 is still supported, this should be the following:
```suggestion
    prompts: List[Union[TextPrompt, TokensPrompt]] = []
```